reaper: 6.12c -> 6.19

ilian opened the pull request:
<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!
List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

###### Motivation for this change
Update Reaper to version 6.19.

Changelog: https://www.reaper.fm/whatsnew.txt

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

ilian committed:
reaper: 6.12c -> 6.19

hugolgst reviewed (commented):
I'd suggest you split c05a9b2 into:
- `maintainers: add ilian`
- `bonzomatic: update maintainers`
- `rpcs3: update maintainers`

Otherwise, the package update looks good to me.

hugolgst commented:
Result of `nixpkgs-review pr 109293` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>reaper</li>
  </ul>
</details>

ilian committed:
maintainers: update username nocent -> ilian

My GitHub ID remains 25505957 as confirmed by the GitHub API:
https://api.github.com/users/ilian

ilian committed:
bonzomatic: update maintainers

ilian committed:
rpcs3: update maintainers

SuperSandro2000 commented:
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 109293` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>reaper</li>
</ul>
</details>

infinisil reviewed (approved):
Looking good! :grinning:

ilian commented:
Thanks for the fast review :+1:
